WordPress/Classes/ViewRelated/Jetpack/Jetpack Scan/JetpackScanCoordinator.swift

```diff
@@ -225,6 +232,20 @@ class JetpackScanCoordinator {
         }
     }
 
+    private func threatsDidChange() {
```

Comment:
This seems very specific to configuring sections. Instead of `threatsDidChange` (which is very generic) can we perhaps use `configureSections` or something similar?

Reply:
Good idea, done in: 4a135ca08409bec9324c3a0ee32f3df7514013f8